### ISSUE TITLE
Problems with confirmation email

### ISSUE THREAD

**Issue by tmtmtmtm** [ciudadanointeligente/write-it#660]
- [x] The first URL is wrong
- [x] The second URL is wrong, but is unnecessary anyway
- [x] Very odd linebreaking throughout
- [x] Do we really need to say who it's to 4 times? (there's also one in the Subject line)

```
Hello Tony


You just submitted a message to

Stubb Alexander

via Eduskunta.


Please visit the following link to confirm you want to send this message

http://writeit.ciudadanointeligente.orghttp://writeit.ciudadanointeligente.org/en/confirm_message/00fd65ced49f11e49114f23c9133f250

(If you can’t click the link, try copying and pasting it into your
browser’s address bar)


Once you have confirmed the message, you can access it by going to

http://writeit.ciudadanointeligente.orghttp://eduskunta-2.writeit.ciudadanointeligente.org/en/thread/test-112/


**IMPORTANT** Once confirmed, this message, will be sent to

Stubb Alexander.

It will also be published on Eduskunta,
where your name, your message, and any replies, will be public and
online for anyone to read, and will also probably appear in search
engine results.


If this message didn’t come from you (or you’ve changed your mind and
don’t want to send it after all) please just ignore this email.


Thanks for using Eduskunta, and
here is a copy of your message for your records:


To: Stubb Alexander
Subject: Test

Test. Please ignore.
```

**Comment by duncanparkes** [comment]
As with #661, use `textwrap` to fix the line breaks.

**Comment by duncanparkes** [comment]
Look at nuntium migrations 57 and 58 and use something like a combination of the two to update existing templates.

**Comment by lfalvarez** [comment]
I think that the urls are fixed in bf607b5b43864d2a922e251437254ad4f90368b7. I believe that the necessity of them is not fixed there though.
